dist/default-passive-events.js

```diff
@@ -1,4 +1,6 @@
 (function() {
+  window.eventListenerOptionsSupported = eventListenerOptionsSupported;
```

Comment:
I think this is out of scope from the library and also don't like the fact of exporting the function to the global scope